psi4/driver/qcdb/dashparam.py

```diff
@@ -309,6 +310,26 @@ def get_dispersion_aliases():
         'pbe0'        : {'s6': 1.000, 's8': 0.528823, 'a1': 0.007912, 'a2': 6.162326},
         'lcwpbe'      : {'s6': 1.000, 's8': 0.906564, 'a1': 0.563761, 'a2': 3.593680},
     },
+    'nl': { # TODO: citations
+        'blyp'        : {'b': 4.000, 'c': 0.0093}, #http://pubs.acs.org/doi/full/10.1021/acs.jctc.5b01066
+        'revpbe'      : {'b': 3.700, 'c': 0.0093}, # and below
+        'b3lyp'       : {'b': 4.800, 'c': 0.0093},
+        'b3pw91'      : {'b': 4.500, 'c': 0.0093},
+        'revpbe0'     : {'b': 4.300, 'c': 0.0093},
+        'bp86'        : {'b': 4.400, 'c': 0.0093},
+        'pbe0'        : {'b': 6.900, 'c': 0.0093},
+        'pbe'         : {'b': 6.400, 'c': 0.0093},
+        'tpss0'       : {'b': 5.500, 'c': 0.0093},
+        'tpss'        : {'b': 5.000, 'c': 0.0093},
+        'dsd-pbepbe'  : {'b': 9.600, 'c': 0.0093},
+        'dsd-pbeb95'  : {'b': 12.500, 'c': 0.0093},
+        'dsd-pbep86'  : {'b': 12.800, 'c': 0.0093},
+        'b2gpplyp'    : {'b': 9.900, 'c': 0.0093},
+        'b2plyp'      : {'b': 7.800, 'c': 0.0093}, # 10.1021/acs.jctc.5b00002
+        'dsd-blyp'    : {'b': 12.000, 'c': 0.0093}, #https://pubs.acs.org/doi/10.1021/ct500642x
```

Comment:
Any of these parametrised in "Hujo, W.; Grimme, S; (2011), J. Chem. Theory Comput.; 7:3866"? If not, I'd add: 
`... "citation": '     citation in std format, as above' },` 
to each of the dicts, so that the `6b) custom citation` section in EmpiricalDispersion picks them up.

Reply:
only partially, I have to dig out the citations.